openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -244,15 +291,21 @@ def _create_render_settings(self):
         system_settings = get_system_settings()["modules"]
 
         deadline_enabled = system_settings["deadline"]["enabled"]
+        royalrender_enabled = system_settings["royalrender"]["enabled"]
         muster_enabled = system_settings["muster"]["enabled"]
         muster_url = system_settings["muster"]["MUSTER_REST_URL"]
 
-        if deadline_enabled and muster_enabled:
+        if deadline_enabled and muster_enabled and royalrender_enabled:
             self.log.error(
-                "Both Deadline and Muster are enabled. " "Cannot support both."
+                ("Multiple render farm support (Deadline/RoyalRender/Muster) "
+                 "is enabled. We support only one at time.")
             )
             raise RuntimeError("Both Deadline and Muster are enabled")
```

Comment:
How about actually mentioning which were enabled now?

```python
farms = ["deadline", "royalrender", "muster"]
enabled_farms = {farm for farm in farms if system_settings.get(farm, {}).get("enabled")}

if len(enabled_farms) > 1:
    message = (
        "Multiple render farm support ({}) is enabled." 
        "We support only one at time.".format("/".join(enabled_farms))
    )
    self.log.error(msg)
    raise RuntimeError(msg)
```

The branches below could become `if "royalrender" in enabled_farms` to use this data too.